test/index.js

```diff
@@ -27,17 +27,34 @@ if (typeof global.gc !== 'function') {
   process.exit(process.exitCode);
 }
 
+const testModules = [];
+
 const fs = require('fs');
 const path = require('path');
 
-let testModules = [];
+let filterCondition = process.env.npm_config_filter || '';
+let filterConditionFiles = [];
+
+if (process.env.npm_config_filter !== null && process.env.npm_config_filter !== undefined && process.env.npm_config_filter.trim() !== '') {
```

Comment:
Why not `if (filterCondition != null && ...)`, accounting for the fact that the default is `''` and not `null`?

Reply:
agreed